sql/moz-fx-data-shared-prod/relay_derived/subscriptions_v1/schema.yaml

```
@@ -0,0 +1,112 @@
+fields:
+- name: customer_id
+  type: STRING
+  mode: NULLABLE
+- name: subscription_id
+  type: STRING
+  mode: NULLABLE
+- name: original_subscription_id
+  type: STRING
+  mode: NULLABLE
+- name: plan_id
+  type: STRING
+  mode: NULLABLE
+- name: status
+  type: STRING
+  mode: NULLABLE
+- name: event_timestamp
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: customer_start_date
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: subscription_start_date
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: original_subscription_start_date
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: subscription_start_reason
+  type: STRING
+  mode: NULLABLE
+- name: trial_start
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: trial_end
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: canceled_at
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: canceled_for_customer_at
+  type: STRING
+  mode: NULLABLE
+- name: cancel_at
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: cancel_at_period_end
+  type: BOOLEAN
+  mode: NULLABLE
+- name: ended_at
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: ended_reason
+  type: STRING
+  mode: NULLABLE
+- name: end_date
+  type: TIMESTAMP
+  mode: NULLABLE
+- name: country
+  type: STRING
+  mode: NULLABLE
+- name: country_name
+  type: STRING
+  mode: NULLABLE
+- name: provider
+  type: STRING
+  mode: NULLABLE
+- name: plan_amount
+  type: INTEGER
+  mode: NULLABLE
+- name: plan_currency
+  type: STRING
+  mode: NULLABLE
+- name: plan_interval
+  type: STRING
+  mode: NULLABLE
+- name: plan_interval_count
+  type: INTEGER
+  mode: NULLABLE
+- name: plan_interval_timezone
+  type: STRING
+  mode: NULLABLE
+- name: product_id
+  type: STRING
+  mode: NULLABLE
+- name: product_name
+  type: STRING
+  mode: NULLABLE
+- name: pricing_plan
+  type: STRING
+  mode: NULLABLE
+- name: billing_grace_period
+  type: INTERVAL
+  mode: REPEATED
+- name: promotion_codes
+  type: STRING
+  mode: NULLABLE
```

The modes for these two got reversed.
```suggestion
- name: billing_grace_period
  type: INTERVAL
  mode: NULLABLE
- name: promotion_codes
  type: STRING
  mode: REPEATED
```

---

done

---

Done.